### ISSUE TITLE
Make Backdrop Console download contrib modules from anywhere within document root

### ISSUE THREAD

**Issue by alanmels** [backdrop-contrib/b#60]
I tried to download a module from within the `modules` directory and it didn't work:

```
docker@cli:/var/www/docroot/modules$ b dl devel
 [x]   Please change directory to BackdropCMS root or specify --root option. 
 [x]   Failed to detect destination directory.                               

```

However, doing the same from the root directory worked fine:

```
docker@cli:/var/www/docroot/modules$ cd ..
docker@cli:/var/www/docroot$ b dl devel
 [v]   devel downloaded to /var/www/docroot/modules/devel! 

```

We are all used to `drush` way of being able to download contributed Drupal modules from within any sub-directory under Drupal setup's main document root, so it would be nice to have the same convenience with Backdrop Console.

**Comment by ghost** [comment]
@alanmels Thanks for posting this. I've noticed the same, and I'm addressing it as a part of my PR for multisite support in https://github.com/backdrop-contrib/b/issues/12. I'm just waiting on getting tests committed so I can update my PR with tests for multisite, then I'll commit that and this will be fixed at the same time.

For the record, my PR makes it so that you can run the download command from any folder in Backdrop. If it's a folder under `/sites/[my-site]` then downloads will go to `/sites/[my-site]/modules`, anywhere else and they'll go to `/modules`.

Closing this in favour of that https://github.com/backdrop-contrib/b/issues/12 then.
